tests/unit/data_formats/datumaro/test_datumaro_format.py

```diff
@@ -126,6 +137,7 @@ def test_can_save_and_load(
             compare=compare,
             require_media=require_media,
             importer_args=fxt_import_kwargs,
+            stream=stream,
         )
```

Comment:
don't we need to test the memory consumption when setting `stream=True`?

Reply:
That is hard to test programmatically. Therefore, I replaced it with testing `is_stream` propagation.